src/SprykerEco/Zed/Computop/Persistence/ComputopQueryContainer.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Persistence;
+
+use Spryker\Zed\Kernel\Persistence\AbstractQueryContainer;
+
+/**
+ * @method \SprykerEco\Zed\Computop\Persistence\ComputopPersistenceFactory getFactory()
+ */
+class ComputopQueryContainer extends AbstractQueryContainer implements ComputopQueryContainerInterface
+{
+    /**
+     * @api
+     *
+     * @param int $idPayment
+     *
+     * @return \Orm\Zed\Computop\Persistence\SpyPaymentComputopQuery
+     */
+    public function queryPaymentById($idPayment)
+    {
+        return $this
+            ->queryPayments()
+            ->filterByIdPaymentComputop($idPayment);
+    }
+
+    /**
+     * @api
+     *
+     * @param int $idOrder
+     *
+     * @return \Orm\Zed\Computop\Persistence\SpyPaymentComputopQuery
+     */
+    public function queryPaymentByOrderId($idOrder)
+    {
+        return $this
+            ->queryPayments()
+            ->filterByFkSalesOrder($idOrder);
+    }
+
+    /**
+     * @api
+     *
+     * @param string $idPay
+     *
+     * @return \Orm\Zed\Computop\Persistence\SpyPaymentComputopQuery
+     */
+    public function queryPaymentByPayId($idPay)
+    {
+        return $this
+            ->queryPayments()
+            ->filterByPayId($idPay);
+    }
+
+    /**
+     * @api
+     *
+     * @param string $idTransaction
+     *
+     * @return \Orm\Zed\Computop\Persistence\SpyPaymentComputopQuery
+     */
+    public function queryPaymentByTransactionId($idTransaction)
+    {
+        return $this
+            ->queryPayments()
+            ->filterByTransId($idTransaction);
+    }
+
+    /**
+     * @param int $orderItemId
+     *
+     * @return \Orm\Zed\Computop\Persistence\SpyPaymentComputopOrderItemQuery
+     */
+    public function queryPaymentItemByOrderItemId($orderItemId)
+    {
+        return $this
+            ->queryPaymentOrderItems()
+            ->filterByFkSalesOrderItem($orderItemId);
+    }
+
+    /**
+     * @api
```

Comment:
`@api` not required for `protected` methods (btw, not all `public` have `@api`…)